website/pages/en/index.js

```diff
@@ -29,6 +29,9 @@ const translate = require('../../server/translate').translate
 class Index extends React.Component {
 	render() {
 		const { config: siteConfig, language = '' } = this.props
+	  const { baseUrl } = siteConfig;
```

Comment:
Needs format

Reply:
Thanks. Have updated all JSs to use spacing for indentation